README.md

```diff
@@ -3,3 +3,14 @@
 ### SDP spring semester 2021
```

Comment:
Don't forget to remove your sprint report from the readme if it has been put in the wiki.